CONTRIBUTING.md

```diff
@@ -1,26 +1,79 @@
 # Contributing
```

Comment:
This might also need the ugly comment to remove it from the TOC.  Although with these documents, once the template is in place, the headings will change so infrequently that manually updating the TOC isn't difficult and saves some of the runaround that the extension gives up.

Reply:
```suggestion
# Contributing <!-- omit in toc -->
```